client/src/components/_common/HighlightSearchTerm/HighlightSearchTerm.scss

```diff
@@ -0,0 +1,3 @@
+.highlight {
+  font-weight: bold;
+}
```

Comment:
**If** we will have this stylesheet*, **then** please make it a CSS module `HighlightSearchTerm.module.css`.

> **Note**
> **Reviewers**, I asked `asimregi` for this arguably-redundant style. I interpret [MDN's article on `<b>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) as implicitly suggesting we **not** rely on browser default (bold) for styling (thus we should explicitly define it here). But, I yield to the opinion of my peers and context here.

Reply:
Updated